anomalib/deploy/inferencers/openvino_inference.py

```diff
@@ -16,12 +16,13 @@
 
```

Comment:
Can we rename this file to `openvino_inferencer.py`? The naming is a bit confusing now, because the file has the same name as `tools/inference/openvino_inference.py`